contrib/Dockerfile.manylinux

```
@@ -97,6 +183,43 @@ ENV OPENSSL_ROOT_DIR="/usr/local/openssl3"
 ENV OPENSSL_LIBRARIES="/usr/local/openssl3/lib64:/usr/local/openssl3/lib"
 ENV OPENSSL_INCLUDE_DIR="/usr/local/openssl3/include"
 
+# --- INFINIA (libred_client) transitive deps not packaged at the right soname on EL8 ---
+# liburing 2.x: EL8 only ships liburing.so.1 (1.0.7); libred_client needs liburing.so.2.
+RUN cd /tmp && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)